app/models/model_file.rb

```diff
@@ -229,6 +229,18 @@ def ctime
     created_at
   end
 
+  def update_from_url!(url:)
+    attachment_attacher.assign_remote_url(
```

Comment:
This is missing a save: see https://github.com/manyfold3d/manyfold/pull/4525 for the fixed version